Refactor: Extract test configuration (#32)
runziggurat/xrpl: SPEC.md

### ZG-CONFORMANCE-006

    The node requests mtGET_PEER_SHARD_INFO_V2 after connection and handshake.

    <>
    <- mtGET_PEER_SHARD_INFO_V2

### ZG-CONFORMANCE-007

>     The node should *NOT* request mtGET_PEER_SHARD_INFO_V2 after connection if no handshake.

Reviewer: ```suggestion
    The node should *NOT* request mtGET_PEER_SHARD_INFO_V2 after connection if there was no handshake.
```

>     The test waits for predefined timeout ensuring no such messages were received.

Reviewer: ```suggestion
    The test waits for the predefined amount of time, ensuring no such messages were received.
```

### ZG-CONFORMANCE-006

    The node requests mtGET_PEER_SHARD_INFO_V2 after connection and handshake.

    <>

>     <-> handshake

Reviewer: ```suggestion
```
`<>` implies a complete handshake

Author: I thought this only implies connection. How to represent 'no handshake'?

Author: Done
